packages/frontend/src/components/UserSettings/ProjectManagementPanel/ProjectManagementPanel.styles.ts

```diff
@@ -44,3 +44,7 @@ export const ProjectTag = styled(Tag)`
     width: fit-content;
     margin-top: 0.3em;
 `;
+
+export const PanelTitle = styled(H5)`
+    margin: 0;
+`;
```

Comment:
Figured that moving this wouldn't hurt. Kept it as it is so that I avoid checking that ProjectManagementPanel looks ok if I replaced it with mantine's `Title` instead @IrakliJani